proposals/new-project-donation-process.md

```diff
@@ -0,0 +1,69 @@
+# KEP-748: Project Donation to Kubeflow
+
+## Summary
+This KEP outlines how community members can donate their project to the Kubeflow community.
+
+## Motivation
+As Kubeflow has become a well established ecosystem and community, several
+projects may want to be donated to the Kubeflow platform to explicitly join our 
+community.
+
+### Goals
+This goal of this process is to give clear guidelines and set expectations 
+for community members about how to be formally included into Kubeflow and 
+the application process.
+
+The decision making process will be separate from the application process and is at the 
+discretion of the Kubeflow Steering Committee. The application and data provided are 
+critical for the KSC to make an informed decision that is best for the longetivity 
+of the project and community.
+
+### Non-Goals
+- Give specific recommendations for evaluating any individual project.
+- Supporting project add-ons.
+
+## Proposal
+The process for donation is intended to be simple but thorough.
+
+Project owners or maintainers will apply for donation by following a four
+step process. 
+
+The process is outlined in six steps:
+
+1. Create a GitHub Issue with a Google Document outlining your proposal (please allow for commentary), the document should have a rough outline with:
+    - Authors
+    - Motivation
+    - Benefits for Kubeflow
+    - Benefits for the Project's Community
+    - Community Metrics
+    - Contributor Metrics
+    - Maintainers
+    - Migration  Plan
+    - Other Related Projects
```

Comment:
Should we also have these:
1. Adoption
2. Licence Agreement
3. Vendor Neutrality
4. Trademark transition
5. CI/CD Infra requirements
6. Current governance structure

Reply:
cc @franciscojavierarceo for the feedback on these points.

Reply:
I think we can put this in the application form.

Reply:
I added all of these except the trademark transition, as I think that's useful information but not a critical detail.

Reply:
If I understand it correctly for the trademarks, since Kubeflow is part of CNCF, the projects should transfer trademark to CNCF if there is any.
But would be nice to get CNCF feedback on this.